### PR TITLE
Fix docs + new workflows

### PR DESCRIPTION
**Pull request by yotamloe**
Fix docs
Add workflow for auto testing for multiple java versions (11 ,13, 15) and operating systems (macos, windows, ubuntu)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +61,8 @@ class MicrometerLogzio {
          @Override
          public String uri() {
             return "<<listener-url>>";
+            // example:
+            // return "https://listener.logz.io:5053";
```
Comment (Doron-Bargo) [inline README.md:65]: return "https://<<listener-url>>:8053";
Also the example should be with port 8053

File: build.gradle [hunk 2]
```diff
@@ -65,49 +65,49 @@ dependencies {
...
-                        organization 'Logz.io'
-                        organizationUrl 'http://logz.io'
-                    }
-                }
-            }
-        }
-    }
-}
\ No newline at end of file
+//uploadArchives {
```
Comment (Doron-Bargo) [inline build.gradle:68]: why we need this code in comment?

**Review by Doron-Bargo — CHANGES_REQUESTED** [review]
Great work
Some cosmetics comments

**Comment by yotamloe** [comment]
changes in this PR are included in #7. closing for now

**Pull request closed without merging**